druntime/src/rt/util/typeinfo.d

```diff
@@ -630,7 +630,7 @@ class TypeInfo_n : TypeInfo
 {
     const: pure: @nogc: nothrow: @safe:
```

Comment:
It already says const here, making all subsequent methods const. Is that insufficient?